### PR TITLE
creating model class, repository, service class, mapper for language …

### PR DESCRIPTION
**Pull request by farhadnajafov**
…table

### CODE REVIEW

**Review by aytiqaqash — APPROVED** [review]
LGTM

File: src/main/java/com/webperside/courseerpbackend/repository/LangugaeRepository.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import java.util.List;
+import java.util.Optional;
+
+@Mapper
+public interface LangugaeRepository {
+    void insert(Language language);
+    List<Language> findAll();
+    Optional<Language> findById(@Param("id") Long id);
+    void update(@RequestBody Language language);
```
Comment (hamidsultanzadeh) [inline src/main/java/com/webperside/courseerpbackend/repository/LangugaeRepository.java:16]: Remove @requestbody from the repository's method

File: src/main/java/com/webperside/courseerpbackend/services/language/LanguageService.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.List;
+
+public interface LanguageService {
+
+    void insert(Language language);
+    List<Language> findAll();
+    Language findById(Long id);
+    void update (@RequestBody Language language);
```
Comment (hamidsultanzadeh) [inline src/main/java/com/webperside/courseerpbackend/services/language/LanguageService.java:13]: same

**Review by hamidsultanzadeh — CHANGES_REQUESTED** [review]
please fix the issues mentioned in comments

**Review by hamidsultanzadeh — APPROVED** [review]

**Review by aytiqaqash — COMMENTED** [review]
LGTM

**Pull request merged**